### DISCUSSION TITLE
new option in OSMand APP = share position live and declare activity type (hike / MTB / hunting...)

### DISCUSSION THREAD

**Discussion by spoirrier** [osmandapp/OsmAnd#13176] [Ideas]
Hello
I have a suggestion to improve nature sharing and reduce risks
users could declare themselves as hiker, MountainBike, Hunter,  and accept geolocalisation
--> then the app would share their live position. and a safety bip could be done by the smartphone if people are too close (example: hiker is less than 50meters from a hunter)

if a lot of people use it, we will make the nature more safe and reduce accidents (hunting, MTB smashing hikers, and so on...
![Capture](https://user-images.githubusercontent.com/94362815/141770551-31e69f4c-4c08-4e78-a4e7-0c309043fdcc.JPG)
)

**Comment by vshcherb** [comment]
Sounds cool!
